pkgs/development/libraries/pinocchio/default.nix

```diff
@@ -13,14 +13,14 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "pinocchio";
-  version = "2.6.20";
+  version = "2.6.21";
 
   src = fetchFromGitHub {
     owner = "stack-of-tasks";
     repo = finalAttrs.pname;
     rev = "v${finalAttrs.version}";
     fetchSubmodules = true;
-    hash = "sha256-Pu/trCpqdue7sQKDbLhyxTfgj/+xRiVcG7Luz6ZQXtM=";
+    hash = "sha256-IaWAXzdzhU/wov+9ChzUeCp9SFNFV2/vGToXC35dkb8=";
   };
 
   # error: use of undeclared identifier '__sincos'
```

Comment:
Looks like `postPatch` here no longer applies?

Reply:
Yes, removing it fixes the package on x86_64-darwin. Since ofborg already passes on other Hydra platforms, I'll push a fix and merge this PR.